card/mod/01_core/set/all/utils.rb

```diff
@@ -28,7 +28,7 @@ def delete_tmp_files_of_cached_uploads
       INNER JOIN cards ON card_actions.card_id = cards.id
       WHERE cards.type_id IN (#{Card::FileID}, #{Card::ImageID}) AND card_actions.draft = true"
     actions.each do |action|
-      if older_than_five_days? action.created_at && card = action.card # we don't want to delete uploads in progress
+      if older_than_five_days?(action.created_at) && card = action.card # we don't want to delete uploads in progress
```

Comment:
Assignment in condition - you probably meant to use `==`.<br>Line is too long. [117/80]